test/distributed/_tensor/test_dtensor_ops.py

```diff
@@ -242,7 +242,6 @@ def wrapped(fn):
     xfail("linalg.vector_norm"),
     xfail("linspace"),
     xfail("log_normal"),
-    xfail("log_softmax"),
     xfail("log_softmax", "with_dtype"),
```

Comment:
hmmm I'm surprised the `with_dtype` overload not passing, we can probably look into it later

Reply:
oh, I didn't test this yet, let me try on top